src/controllers/bundler.controller.ts

```diff
@@ -0,0 +1,40 @@
+import { NextFunction, Request, Response, Router } from 'express';
+import { ProblemException } from '../exceptions/problem.exception';
+import { Controller } from '../interfaces';
+import bundler from '@asyncapi/bundler';
+import Document from '@asyncapi/bundler/lib/document.js';
+import { documentValidationMiddleware } from '../middlewares/document-validation.middleware';
+import path from 'path';
+
+export class BundlerController implements Controller {
+  public basepath = '/bundle';
+  private async bundle(req: Request, res: Response, next: NextFunction) {
+    const asyncapis: Array<string> = req.body.asyncapis || [req.body.asyncapi];
+    if (!asyncapis || !asyncapis.length) {
+      throw new ProblemException({
+        type: 'https://api.asyncapi.com/problem/asyncapi-missing',
+        title: 'Asyncapi missing',
+        detail: 'No Asyncapi provided',
+        status: 400,
+      });
+    }
```

Comment:
`asyncapis` field is checked using middleware that checks the request shape based on OpenAPI definitions, this is not needed, and `asyncapis` always be as array :)

```suggestion
```